task/thermal/src/control.rs

```diff
@@ -226,6 +226,7 @@ pub struct ThermalSensorErrors {
 }
 
 impl ThermalSensorErrors {
+    #[allow(dead_code)]
```

Comment:
This is unrelated to the rest of this PR, it just cleans up a warning currently on `master`.